tests/spirv/CMakeLists.txt

```
@@ -25,7 +25,9 @@
 
 if(ENABLE_SPIRV)
 
-add_compile_options( -Wno-deprecated -Wno-deprecated-declarations)
+if(not MSVC)
```

```suggestion
if(NOT MSVC)
```
I think the CIs are tripping on this one.